ansible/roles/couchdb/templates/couchdb.bin.j2

```diff
@@ -21,7 +21,7 @@ ERL_OS_MON_OPTIONS="-os_mon \
     disk_space_check_interval 1 \
     disk_almost_full_threshold 1"
 ERL_START_OPTIONS="$ERL_OS_MON_OPTIONS -sasl errlog_type error +K true +A 4"
-HEART_BEAT_TIMEOUT=11
+HEART_BEAT_TIMEOUT=120
```

Comment:
maybe there's a better way to do this by updating this script and pulling from a different location but for now i wanted to get this out. there's no way i could find to configure this param out of the box